webui/tests/cypress/support/commands.js

```diff
@@ -38,22 +38,14 @@ Cypress.Commands.add('checkLoggedIn', (username = null, visit = true) => {
   // cluster page link in the top bar, which should be present if and only if
   // the user is logged in.
   username = username || ACCOUNT_USERNAME;
-  if (visit) cy.visit('/');
+  if (visit) cy.visit('/det');
```

Comment:
we shouldn't rely on / redirecting to /det unless we're testing that (which we do in navigation.spec)